<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choice</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(72, 60, 70);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>30</y>
     <width>470</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(72, 60, 70);
color: rgb(255, 255, 255);
border-radius: 10px;
background-color: rgb(30, 25, 29);
background-color: rgb(52, 43, 50);</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Системы Пентагона защищены головоломкой с тремя уровнями сложности. Выберите, какой &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;из них вы хотите пройти:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="level1_bt">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>132</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	border-radius: 10px;
	background-color: rgb(52, 43, 50);
}
QPushButton:pressed{
	background-color: rgb(40, 33, 38);
	color: rgb(65, 255, 62);
}</string>
   </property>
   <property name="text">
    <string>•</string>
   </property>
  </widget>
  <widget class="QPushButton" name="level2_bt">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>132</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	border-radius: 10px;
	background-color: rgb(52, 43, 50);
}
QPushButton:pressed{
	background-color: rgb(40, 33, 38);

	color: rgb(255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string>••</string>
   </property>
  </widget>
  <widget class="QPushButton" name="level3_bt">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>132</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	border-radius: 10px;
	background-color: rgb(52, 43, 50);
}
QPushButton:pressed{
	
	background-color: rgb(40, 33, 38);
	color: rgb(218, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>•••</string>
   </property>
  </widget>
  <widget class="QLabel" name="level1_lb">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>215</y>
     <width>132</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1 уровень:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="level2_lb">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>215</y>
     <width>132</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2 уровень:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="level3_lb">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>215</y>
     <width>132</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3 уровень:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
